rust-src/concordium_base/src/common/types.rs

```diff
@@ -186,6 +187,91 @@ impl Deserial for OwnedParameter {
     }
 }
 
+/// A ratio between two `u64` integers.
+/// It should be safe to assume the denominator is not zero and that numerator
+/// and denominator are coprime.
```

Comment:
Could you also explain why we aren't just using the regular Ratio type here?

Reply:
I have added to the doc comment